DataManagementSystem/DB/FTS3DB.py

```diff
@@ -0,0 +1,456 @@
+__RCSID__ = "$Id $"
+
+# pylint: disable=no-member
```

Comment:
For what?

Reply:
All the class attributes generated by SQLAlchemy

Reply:
I added a comment